courses/free-courses-en.md

```diff
@@ -267,6 +267,7 @@
 
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Google Cloud Certification Course](https://www.youtube.com/watch?v=UGRDM86MBIQ) - Free Code Camp
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Reply:
This is a 6hr video for the complete course theory + code along. So can be considered as a playlist in itself. Name of producer and creator will be added after you review this comment. @eshellman

Reply:
It's ok for me too. 6h video but without chapters integrated in timeline